### PR TITLE
[New Term] Python Built-in Functions: `help()`

### PR DESCRIPTION
**Pull request by alenn97**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

-  [x] All writings are my own.
-  [x] My entry follows the Codecademy Docs style guide.
-  [x] My changes generate no new warnings.
-  [x] I have performed a self-review of my own writing and code.
-  [x] I have checked my entry and corrected any misspellings.
-  [x] I have made corresponding changes to the documentation if needed.
-  [x] I have confirmed my changes are not being pushed from my forked `main` branch.
-  [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
-  [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dusch4593** [comment]
Amazing! Thank you for submitting this, @alenn97 🎉

Either myself or one of my colleagues will step fourth to perform a review for this PR. After at least 2 reviews have been approved, we should be ready to merge this.

One last piece of advice I'd like to offer regarding your Docs fork: Since new changes are constantly being merged into the `main` branch of the base `Codecademy/docs` repo, this will sometimes put your fork's `main` branch x-commits behind:

Screen Shot 2023-01-11 at 11 48 39 AM

Before starting a new PR branch for next entry, you'll need to make sure this is caught up on GitHub, like so:

Screen Shot 2023-01-11 at 11 49 54 AM

Then, on your terminal, go back to your `main` branch with `git checkout main` and then run `git pull` to catch things up on the local end.

**Comment by alenn97** [comment]
Thank you.
If the update is needed for this pull request I will do it as soon as I can.

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/help/help.md:36]: It would be better if the output was presented in a separate ```shell block rather than as comments in the code. Just preface it with a sentence: This example produces the following output:

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/help/help.md:55]: Same comment here. Separate the output into a ```shell block.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@alenn97 I did a first review and only had a couple of comments.

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/help/help.md:38]: There need to be line breaks before and after code blocks

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/help/help.md:57]: There need to be line breaks before and after code blocks.

**Review by SSwiniarski — APPROVED** [review]
@alenn97 aside from a couple of comments about spacing, this looks ready for a second review.

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (alenn97) [reply content/python/concepts/built-in-functions/terms/help/help.md:38]: Should I commit a new version with the correct spacing right now or do I have to wait for the second review?

Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/help/help.md:38]: Go ahead and commit a new version.

**Comment by Dusch4593** [comment]
@alenn97 I'm going to take a look for review 2️⃣ 😄

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/help/help.md:34]:
```suggestion
The following shows how the `help()` function provides information about Python's built-in [`print()`](https://www.codecademy.com/resources/docs/python/built-in-functions/print) function:
```

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/help/help.md:77]:
```suggestion

The following example is runnable and shows how the `help()` function can be applied to different kinds of objects, including user-defined classes:

```

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (Dusch4593) [reply content/python/concepts/built-in-functions/terms/help/help.md:57]: @alenn97 after applying these line breaks, I would then suggest we switch the two examples in this "Example" section. So, the first example should be `help()` + start screen, followed by `help(print)` + documentation results.

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/help/help.md:78]:
```suggestion
```codebyte/python
```

File: content/python/concepts/built-in-functions/terms/help/help.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/help/help.md:9]:
```suggestion
  - 'Documentation'
  - 'Functions'
  - 'Methods'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@alenn97 left you some comments/suggestions for review 2️⃣ 

Let us know if you have any questions or need any assistance with implementing the feedback. 😄

**Review by Dusch4593 — APPROVED** [review]
@alenn97 This looks good to merge! I made the following edits in 16998697d3d361288555e7e4d72ea6c13c91d84f before doing so:

- Combined lines # 24 and 25 into one sentence (now just line # 24).
- Small edits on line # 26 and # 30.
- Removed extra spacing to get the GitHub checks to pass.
-

**Pull request merged**
